mappings/net/minecraft/entity/ai/pathing/PathNodeNavigator.mapping

```diff
@@ -0,0 +1,20 @@
+CLASS none/bad net/minecraft/entity/ai/pathing/PathNodeNavigator
+	FIELD a minHeap Lnone/azy;
+	FIELD b unusedSet Ljava/util/Set;
```

Comment:
If it's unused then don't map it? Unless this is "unused" as in unused nodes or something, doubt it though because then it should probably be `unusedNodes` or something.

"unused" could be another type of marking: FabricMC/Enigma#27 @Thog